batchflow/models/tf/encoder_decoder.py

```diff
@@ -326,3 +329,58 @@ def decoder(cls, inputs, name='decoder', **kwargs):
                         x = cls.crop(x, inputs[-i-3], data_format=kwargs.get('data_format'))
                         x = tf.concat((x, inputs[-i-3]), axis=axis, name='skip-concat')
         return x
+
+
+class AutoEncoder(EncoderDecoder):
+    """ Model without skip-connections between corresponding stages of encoder and decoder. """
+    @classmethod
+    def default_config(cls):
+        config = EncoderDecoder.default_config()
+        config['body/decoder'] = dict(skip=False)
```

Comment:
Some defaults for the embedding block might be useful here, what do you think?

Reply:
Is there any particular difference why it should be included only for autoencoders and not for encoder-decoder in general?
The reason why I did not include that in `EncoderDecoder` class is that it looks rather strange to compose encoder from massive blocks (e.g. DenseNet.block) and use small conv_block as the final encoding step.
Also, default value for number of channels is definitely ill-advised due to it's restrictive nature. For autoencoders in particular, dimensionality of compressed image is one of the most important parameters to control manually.
On the other hand, there is a possibility for not so restrictive default value of not changing number of channels, but it need to be incorporated in the `embedding` method:
```suggestion
        filters = kwargs.get('filters') or num_filters(inputs)
```

But such addition poses a new question: why should not we do the same for encoder/decoder `base` blocks?

Reply:
Added `default_block` in the `EncoderDecoder` class. That is a shape-preserving  `conv_block` with predefined `layout`.
That block is used instead of `conv_block` as default.